<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Conway's Game of Life</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <widget class="QMenu" name="menuRunSettings">
     <property name="title">
      <string>Run Settings</string>
     </property>
     <widget class="QMenu" name="menuSpeed">
      <property name="title">
       <string>Speed</string>
      </property>
     </widget>
     <widget class="QMenu" name="menuThreadSettings">
      <property name="title">
       <string>Thread Settings</string>
      </property>
      <addaction name="actionThreadCount"/>
      <addaction name="separator"/>
      <addaction name="actionUseQtConcurrent"/>
      <addaction name="actionUseQThreads"/>
      <addaction name="separator"/>
     </widget>
     <addaction name="actionUseThreads"/>
     <addaction name="menuThreadSettings"/>
     <addaction name="separator"/>
     <addaction name="menuSpeed"/>
     <addaction name="actionDisplay"/>
     <addaction name="actionFastest"/>
    </widget>
    <widget class="QMenu" name="menuSettings">
     <property name="title">
      <string>Settings</string>
     </property>
     <addaction name="actionShowColours"/>
    </widget>
    <addaction name="actionNew"/>
    <addaction name="actionRandomize"/>
    <addaction name="separator"/>
    <addaction name="menuSettings"/>
    <addaction name="separator"/>
    <addaction name="menuRunSettings"/>
    <addaction name="actionRun"/>
    <addaction name="actionPause"/>
    <addaction name="actionStep"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionRun">
   <property name="text">
    <string>Run</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="actionStep">
   <property name="text">
    <string>Step</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="text">
    <string>Pause</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionRandomize">
   <property name="text">
    <string>Randomi&amp;ze</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionDisplay">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Display</string>
   </property>
  </action>
  <action name="actionFastest">
   <property name="text">
    <string>Fastest</string>
   </property>
  </action>
  <action name="actionShowColours">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Colours</string>
   </property>
  </action>
  <action name="actionUseQtConcurrent">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Use QtConcurrent</string>
   </property>
  </action>
  <action name="actionUseQThreads">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Use QThreads</string>
   </property>
  </action>
  <action name="actionUseThreads">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Use Threads</string>
   </property>
  </action>
  <action name="actionThreadCount">
   <property name="text">
    <string>Thread Count</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
